### PR TITLE
ie11webdriver: Update to version 3.150.1

### PR DESCRIPTION
**Pull request by linsui**
https://github.com/ScoopInstaller/Main/issues/150
[Here](https://github.com/SeleniumHQ/selenium/tree/master/cpp/iedriverserver) is the source code.

### CODE REVIEW

File: bucket/ie11webdriver.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-            "hash": "29de31f5a3028224e5ae2df5757e112755ef8ccd0a06de8f333b129139647d70"
+            "url": "https://selenium-release.storage.googleapis.com/3.150/IEDriverServer_Win32_3.150.1.zip",
+            "hash": "09263561924a5a222b39bd0a80ef3778b8bf45457e1cf91c79b6a9f65d6230aa"
         }
     },
+    "bin": "IEDriverServer.exe",
     "checkver": {
-        "url": "http://www.seleniumhq.org/download/",
-        "re": "Download version ([\\d.]+).*[^<]+32 bit Windows IE"
+        "url": "https://selenium.dev/downloads/",
```
Comment (Ash258) [inline bucket/ie11webdriver.json]: Why you do not check it from raw https://github.com/SeleniumHQ/selenium/blob/master/cpp/iedriverserver/CHANGELOG ?

Comment (linsui) [reply bucket/ie11webdriver.json]: Good idea. 👍

Comment (linsui) [reply bucket/ie11webdriver.json]: Ping. When revision version is 0 we need to drop it. Also we don't know if we should drop the build version when it's 0. It's difficult to get the right version.

Comment (rasa) [reply bucket/ie11webdriver.json]: Is this concern resolved?

**Pull request merged**